Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
Error: styles were not requested.
diff --git a/output/Checkbox Group with Validation Error/1/9e434a6c-2327-4f93-af8d-fcbba76266e1.html b/output/Checkbox Group with Validation Error/1/9e434a6c-2327-4f93-af8d-fcbba76266e1.html
index 5ec8ac9..6786e24 100644
--- a/output/Checkbox Group with Validation Error/1/9e434a6c-2327-4f93-af8d-fcbba76266e1.html	
+++ b/output/Checkbox Group with Validation Error/1/9e434a6c-2327-4f93-af8d-fcbba76266e1.html	
@@ -1,15 +1,16 @@
-<form>
-  <fieldset>
-    <legend>What fruits do you like?</legend>
-    <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-    <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-    <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
-    <div class="error">You must choose one or more fruits</div>
-  </fieldset>
-</form>
-<style>
-  .error {
-    color: red;
-    display: none;
-  }
-</style>
\ No newline at end of file
+<fieldset aria-describedby="fruits-error">
+  <legend>What fruits do you like?</legend>
+  <div class="error" id="fruits-error">You must choose one or more fruits</div>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas" aria-invalid="true">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges" aria-invalid="true">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples" aria-invalid="true">
+    Apples
+  </label>
+</fieldset>
\ No newline at end of file
